### ISSUE TITLE
Need for a new font-patcher script

### ISSUE THREAD

**Issue by mhartington** [ryanoasis/nerd-fonts#97]
#### Subject of the issue
Patching fonts with ligatures
_Describe issue here_
With ligatures kind of being a more common thing, it's good idea to support them. Iterm 2 nightly has support for ligatures, but it needs to be whitelisted.  There might be a need for a new font-patch script that doesn't modify the fonts name and postscript meta data.  So once the fonts get generated, it can be consumed by iterm without any issues.

```

	PostScript name	HaskligNerdFontPlusFontAwesomePlusOcticonsPlusPomicons---Regular
	Full name	Hasklig Nerd Font Plus Font Awesome Plus Octicons Plus Pomicons
	Family	Hasklig Nerd Font
	Style	Regular
	Kind	OpenType PostScript
```

Not sure what the licensing would be for this. It may be that the script would need to be added but not the fonts. I assume you're renaming these fonts for licensing reasons.

See [this line here](https://github.com/gnachman/iTerm2/blob/master/sources/PTYFontInfo.m#L38)
#### Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_

any font with ligatures 😁 
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_

iterm 2 nightly
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_

macOS beta

Let me know if there's anymore info/help needed.

**Comment by ryanoasis** [comment]
@mhartington Hey thanks, yeah I agree it would be cool and even useful to support ligatures. By the way related to this would be #57 

--

I know this repo has been a bit quiet lately but yes I would and do appreciate any help or effort into improving. 

E.g. there has been a fairly substantial alternate patching proposal in #83 but I just haven't gotten to trying it out or providing much thought :cry: 

from @be5invis:
> This script preserves all metadata from the original font, therefore it shares the compatibility of the original font.

--

> I assume you're renaming these fonts for licensing reasons.

Yes exactly the current reason :smiling_imp: 

--

So in summary I am totally for getting ligatures supported or at the very least **not** be butchered by the current patcher

**Comment by mhartington** [comment]
Very good! 😁 

I shall move the talk over to those other issues and try to make a new PR
